<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateStatusDialog</class>
 <widget class="QDialog" name="UpdateStatusDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Status of Defect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="defectListComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Select Status:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="statusComboBox">
     <item>
      <property name="text">
       <string>Open</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>In Progress</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Closed</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
